config.go

```diff
@@ -259,6 +263,36 @@ func fileExists(name string) bool {
 	return true
 }
 
+// validate pub vote and fee keys as belonging to the network
+func (c *config) parsePubKeys(params *chaincfg.Params) error {
+	// Parse the extended public key and the pool fees.
+	var err error
+	coldWalletFeeKey, err = hdkeychain.NewKeyFromString(c.ColdWalletExtPub)
+	if err != nil {
+		return fmt.Errorf("cold wallet extended public key: %v", err)
+	}
+	if !coldWalletFeeKey.IsForNet(params) {
+		return fmt.Errorf("cold wallet extended public key is for wrong network")
+	}
+	// Parse the extended public key for the voting addresses.
+	votingWalletVoteKey, err = hdkeychain.NewKeyFromString(c.VotingWalletExtPub)
+	if err != nil {
+		return fmt.Errorf("voting wallet extended public key: %v", err)
+	}
+	if !votingWalletVoteKey.IsForNet(params) {
+		return fmt.Errorf("voting wallet extended public key is for wrong network")
+	}
+	return nil
+}
+
+func (c *config) GetFeeKey() *hdkeychain.ExtendedKey {
```

Comment:
Style issue: https://golang.org/doc/effective_go.html#Getters
We try to follow this in general

Reply:
But these getters seem unneeded.  `coldWalletFeeKey` is already accessible by main (same package).  Also, the receiver of this getter is pointless.

Reply:
Thank you Chapp! I will start reading through effective go.

I didn't even realise they were the same package.  Still learning, thank you.

Is this better?